### PR TITLE
chore: remove unused deps

### PR DESCRIPTION
**Pull request by KenLSM**
## Problem

Removal of unused deps avoid noise from dependabots and false positives from security scanners.

These removals also improves the build times by about 30s due to having smaller `node_modules`. Biggest contributor is from puppeteer since it also installs a copy of chromium.

## Solution

Removals:
- `puppeteer`: Headless browser, unused. <check why this is installed>
- `worker-loader`: Unused. Previously in `webpack / craco`, but since we've moved to `vite`, there's no need for this loader
- `use-debounce`: Completely unused. Prefer `react-use`
- `nocache`: Unused in BE code. Removed in tests.
- `webpack*`: Unused since we're using `vite`
- `regenerator`: Unused since we're using `vite`

**Breaking Changes** 

- No - this PR is backwards compatible  

## Tests

**Regression**
#### Landing page renders
- [ ] Ensure that landing page renders

### CODE REVIEW

**Comment by LoneRifle** [comment]
If you're removing webpack for good, could I kindly ask to take the trouble to update the README so that the user knows to visit `localhost:5173` now? Thanks =)

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditUen.tsx [hunk 1]
```diff
@@ -46,6 +46,9 @@ export const EditUen = ({ field }: EditUenProps): JSX.Element => {
 
   return (
     <CreatePageDrawerContentContainer>
+      {/* TESTING CODE TO TRIGGER STORYBOOK */}
```
Comment (scottheng96) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditUen.tsx]: Should this be removed after testing it

**Comment by KenLSM** [comment]
> If you're removing webpack for good, could I kindly ask to take the trouble to update the README so that the user knows to visit `localhost:5173` now? Thanks =)

@littlemight has already ahead on this :) https://github.com/opengovsg/FormSG/pull/7930

**Review by scottheng96 — APPROVED** [review]
Looks good, only leftover to remove that

**Comment by KenLSM** [comment]
Screenshot 2024-11-22 at 1 12 13 PM
Seemed to be stuck on "Processing updates" for a couple of hours now. Gonna manually squash to see if it helps.

**Pull request merged**